frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -123,11 +113,34 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)<{
     `}
 `;
 
-export const CardsContainer = styled(FullWidthContainer)`
-  margin-top: 8px;
+export const ParametersAndCardsContainer = styled.div<{
+  shouldMakeDashboardHeaderStickyAfterScrolling: boolean;
+}>`
+  flex: auto;
+  min-width: 0;
+  overflow-y: ${({ shouldMakeDashboardHeaderStickyAfterScrolling }) =>
+    shouldMakeDashboardHeaderStickyAfterScrolling ? "auto" : "visible"};
+  overflow-x: hidden;
+  @supports (overflow-x: clip) {
+    overflow-x: clip;
+  }
+  padding-bottom: 40px;
 
   &.${SAVING_DOM_IMAGE_CLASS} {
-    padding-bottom: 20px;
+    ${ParametersWidgetContainer} {
+      background-color: transparent;
+      border-bottom: none;
+
+      margin-top: 1rem;
+
+      legend {
+        top: -12px;
+      }
+    }
```

Comment:
Workaround: `html2canvas` doesn't handle `fieldset` + `legend` elements we're using for filters
https://github.com/niklasvh/html2canvas/issues/2418

![cleanshot_2024-02-21_at_17 03 30_2x](https://github.com/metabase/metabase/assets/17258145/c1dbc3c2-273e-424a-887f-153f517e01d2)